src/frontend/Components/Thumbnail/Thumbnail.css

```diff
@@ -0,0 +1,11 @@
+.thumbnail-container{
+    border: 1px solid black;
+    width: 12rem;
```

Comment:
Please try to use % when using width as this makes is more flexible when thinking of responsiveness.